scripts/error-codes/codes.json

```diff
@@ -0,0 +1,185 @@
+{
```

Comment:
Yeah, this is from the commit that you added. The new errors comes from running `grunt extract-errors`.